spec/services/amalgamator/processor/base_spec.rb

```diff
@@ -39,7 +39,7 @@
 
     it 'rounds values to 8 decimals and replaces nil with 0.0' do
```

Comment:
If we have changed the method from `round` to `floor`, shouldn't we update the comment as well?